packages/components/src/FormField/components/ClearAction.css

```diff
@@ -12,6 +12,7 @@
   cursor: pointer;
   justify-content: center;
   align-items: center;
+  transform: translateY(50%);
```

Comment:
![Screenshot 2024-10-03 at 11 52 13 AM](https://github.com/user-attachments/assets/1a025d6c-faac-4151-837b-3e238994368e)

Added this back as well (it was removed in initial PR) to avoid clearable being at the top of the input